doc/jedi-vim.txt

```diff
@@ -25,10 +25,11 @@ Contents                                *jedi-vim-contents*
     5.2. Go to definition               |g:jedi#goto_command|
     5.3. Go to assignment               |g:jedi#goto_assignments_command|
     5.4  Go to definition (deprecated)  |g:jedi#goto_definitions_command|
-    5.5. Show documentation             |g:jedi#documentation_command|
-    5.6. Rename variables               |g:jedi#rename_command|
-    5.7. Show name usages               |g:jedi#usages_command|
-    5.8. Open module by name            |:Pyimport|
+    5.5  Go to stub                     |g:jedi#goto_stubs_command|
+    5.6. Show documentation             |g:jedi#documentation_command|
+    5.7. Rename variables               |g:jedi#rename_command|
+    5.8. Show name usages               |g:jedi#usages_command|
+    5.9. Open module by name            |:Pyimport|
```

Comment:
Replace 5.4 ("Go to definition (deprecated)") with the new one?  (less diff churn, and phasing out the deprecated thing).

Reply:
I pushed something.